src/main/java/org/opentripplanner/routing/api/request/RoutingRequest.java

```diff
@@ -1426,11 +1378,12 @@ public void setWaitAtBeginningFactor(double waitAtBeginningFactor) {
         return bannedRoutes;
     }
 
-    public double getMaxAccessEgressDurationSecondsForMode(StreetMode mode) {
-        return maxAccessEgressDurationSecondsForMode.getOrDefault(
+    public Duration getMaxAccessEgressDuration(StreetMode mode) {
```

Comment:
I made a mistake when renaming this method. Should we put the `forMode` part back?